### PR TITLE
feat: add a news article about the Baby Bliss Bot project

### PR DESCRIPTION
**Pull request by cindyli**
@agamba, could you review this news article. Pinging you here as your github id is not showing up in the reviewers list.

### CODE REVIEW

File: src/collections/news/2023-04-12-BBB-Intro.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The origin of the use of Bliss as a bridge to communication and literacy can be traced back to the
+pioneer program initiated by Shirley McNaughton at the Ontario Crippled Children's Centre in 1971.
+Blissymbols offer a written semantic language without phonology that serves as a valuable alternative
+for nonvocal individuals, such as those with cerebral palsy, who face obstacles in acquiring phonological
+language skills.
+
+The rapid advancement of AI technology, particularly Large Language Models (LLMs), presents both a
+tremendous opportunity to enhance societal engagement for nonvocal citizens but also comes with many
+risks for a group that is not recognized or included in the training data. The Baby Bliss Bot (BBB)
+project embarks on a community-led investigation to explore the possibilities and challenges of
```
Comment (klown) [inline src/collections/news/2023-04-12-BBB-Intro.md:24]: Suggest replacement for "embarks on" in "The Baby Bliss Bot (BBB) project embarks on a community-led investigation ..."

Instead:  "The Baby Bliss Bot (BBB) project is a community-led investigation ..."

**Review by klown — CHANGES_REQUESTED** [review]

**Comment by cindyli** [comment]
@klown thanks for the comment. It has been addressed and ready for another look.

**Review by klown — APPROVED** [review]
Look good to me @cindyli.

File: src/collections/news/2023-04-12-BBB-Intro.md [hunk 2]
```diff
@@ -2,25 +2,28 @@
...
-project embarks on a community-led investigation to explore the possibilities and challenges of
-leveraging AI to improve communication among Bliss users.
+The Baby Bliss Bot project delves into the realm of using AI tools for language development
+and translation to enhance communication for AAC users who rely on a minority language system.
+This article examines the background of Blissymbolics, the potential of AI technology, and
+the objectives of the project.
+
+[Blissymbolics](https://www.blissymbolics.org/), a universal symbol language originally developed
+by Charles K. Bliss for the purpose of international communication, has gained popularity as a
+method for teaching communication to children with physical disabilities who are non-speaking.
```
Comment (agamba) [inline src/collections/news/2023-04-12-BBB-Intro.md:14]: I suggest rephrasing 
"has gained popularity as a method for teaching communication to children with physical disabilities who are non-speaking."
by
"has gained popularity as a method for assisting language development and communication skills to children with physical disabilities who are non-speaking."

File: src/collections/news/2023-04-12-BBB-Intro.md [hunk 3]
```diff
@@ -2,25 +2,28 @@
...
+The origin of the use of Bliss as a bridge to communication and literacy can be traced back to the
+pioneer program initiated by Shirley McNaughton at the Ontario Crippled Children's Centre in 1971.
+Blissymbols offer a written semantic language without phonology that serves as a valuable alternative
+for nonvocal individuals, such as those with cerebral palsy, who face obstacles in acquiring phonological
+language skills.
+
+The rapid advancement of AI technology, particularly Large Language Models (LLMs), presents both a
+tremendous opportunity to enhance societal engagement for nonvocal citizens but also comes with many
```
Comment (agamba) [inline src/collections/news/2023-04-12-BBB-Intro.md:22]: Not sure if we want to make the statement refer to "nonvocal citizens" or more broadly to "nonvocal individuals"

**Comment by cindyli** [comment]
Thanks for comments, @agamba. I've addressed them all and pushed to the main branch.

**Pull request merged**